### ISSUE TITLE
"read" event never fired to api.listen

### ISSUE THREAD

**Issue by Darkfafi** [Schmavery/facebook-chat-api#501]
api.listen((err, message) => {
    	console.log(message.type);

**Comment by bsansouci** [comment]
Hey! Have you tried turning events on using `api.setOptions({listenEvents: true})` before listen?

**Comment by Darkfafi** [comment]
Yes, also tried that. Did not help... Sorry for the late response btw. Was out of the country for some time.

**Comment by ravkr** [comment]
is this still a problem? or can we close?

**Comment by Bjornskjald** [comment]
Sorry for bumping such an old thread, but... no events work for me on any account I manage.

**Comment by Schmavery** [comment]
Any events at all? Good to know. Anyone else seeing this?
I wonder if this is a different issue than the original (which seems to just be reads). Nevertheless, feel free to do some digging around and maybe we can figure it out!

**Comment by Bjornskjald** [comment]
@Schmavery I'd love to do some testing, can I enable some debug output or anything?

**Comment by Schmavery** [comment]
You can set the loglevel in setOptions. Something like this should work:
```
api.setOptions({
    logLevel: "verbose"
});
```
